src/plugins/tabs/components/tabs.vue

```diff
@@ -153,6 +148,39 @@ export default {
 
             this.currentTab = tab;
         },
+        showRemoveDialog(index) {
+            if(!this.tabsList[index]) {
+                return;
+            }
+
+            let activeTab = this.tabsList[index];
+            let childDomainObject = activeTab.domainObject
+
+            let self = this;
+            let prompt = this.openmct.overlays.dialog({
+                iconClass: 'alert',
+                message: `This action will remove this tab from the Tabs Layout. Do you want to continue?`,
+                buttons: [
+                    {
+                        label: 'Ok',
+                        emphasis: 'true',
+                        callback: function () {
```

Comment:
we can use es6 Arrow functions here, this way we dont have to reassign this to self

Reply:
Code updated to use ES6 arrow funcs.